com.unity.render-pipelines.high-definition/Documentation~/Custom-Pass-Creating.md

```diff
@@ -463,6 +463,11 @@ You can also load custom buffers using the following functions:
 - `LoadCustomColor(uint2 pixelCoords)`
 - `LoadCustomDepth(uint2 pixelCoords)`
 
+Note that depending on the injection point used for the Fullscreen custom pass, sampling the custom buffer can result in incorrect scaling. Thus it's recommended to use these functions instead:
```

Comment:
Which injection points cause incorrect scaling? I think it would be helpful to the user to be specific here.

Reply:
These injection points were causing scaling issues:
- BeforePreRefraction
- BeforePostProcess
- AfterPostProcess